apps/src/code-studio/pd/workshop_dashboard/components/date_picker.story.jsx

```diff
@@ -3,21 +3,30 @@ import moment from 'moment';
 import DatePicker from './date_picker';
 import {action} from '@storybook/addon-actions';
 
-export default storybook => {
-  return storybook
-    .storiesOf('DatePicker', module)
-    .addDecorator(story => (
-      // Currently the Bootstrap 3 styles required by React-Bootstrap are only applied inside div#workshop-container.
-      // This is to prevent conflicts with other parts of Code Studio using Bootstrap 2.
-      // See pd.scss. Without this container div it won't render properly.
-      <div id="workshop-container" style={{width: 300}}>
-        {story()}
-      </div>
-    ))
-    .add('Basic', () => (
-      <DatePicker date={moment()} onChange={action('changed')} />
-    ))
-    .add('Clearable', () => (
-      <DatePicker date={moment()} onChange={action('changed')} clearable />
-    ));
+export default {
+  title: 'WorkshopDashboard/DatePicker',
```

Comment:
I think in the past we've tried to keep this structure flat, though I don't have a strong opinion here. If other components are also using this prefix, I think this is fine!

Reply:
I wasn't sure about this, and also don't have a strong opinion so I'll remove the `WorkshopDashboard` bit!